Stop executing YAML tags for mkdocs_theme.yml, warn about third-party projects (#3465)
mkdocs/mkdocs: docs/dev-guide/plugins.md

pip install mkdocs-foo-plugin
```

> WARNING: Installing an MkDocs plugin means installing a Python package and executing any code that the author has put in there. So, exercise the usual caution; there's no attempt at sandboxing.

Reviewer: ```suggestion
WARNING: By installing an MkDocs plugin you are installing a Python package and executing any code embedded by the author. There's no attempt at sandboxing, so exercise the usual caution.
```

Author: Sorry, I like my wording better :s
